Tools/Scripts/build-webkit

```diff
@@ -240,6 +240,27 @@ if (isAppleCocoaWebKit()) {
 
 my $result = 0;
 
+if (isGtk() || isWPE()) {
```

Comment:
I think this would be useful also for jsc-only builds, so i suggest to put the code inside the `if (isCMakeBuild() && !isAnyWindows())` block below that way it will enable for gtk, wpe and jsc-only (on linux)